src/chrome/content/rules/Geekzone.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="geekzone.co.nz (partial cdn)">
+	<target host="cdn.geekzone.co.nz" />
+	<target host="images.geekzone.co.nz" />

```

Comment:
You can use regex in the `rule` and `exclusion` tags. If it's not worth to make complex patterns, because deep links work only randomly over https, I would just declare it as non-working.